litho-core/src/main/java/com/facebook/litho/LogTreePopulator.java

```diff
@@ -38,6 +38,21 @@ private LogTreePopulator() {}
   @CheckReturnValue
   public static PerfEvent populatePerfEventFromLogger(
       ComponentContext c, ComponentsLogger logger, PerfEvent perfEvent) {
+    return populatePerfEventFromLogger(c, c.getTreeProps(), logger, perfEvent);
+  }
+
+  /**
+   * Annotate a log event with the log tag set in the context, and extract the treeprops from a
```

Comment:
Seems that jdoc was just copy-pasted without modification

Reply:
Thanks Sergey, updated.